01-arch/04-object_split.md

```diff
@@ -6,22 +6,27 @@ For each part of the original object's payload, a separate object with own `Obje
 
 ![Large object split](pic/object_split_all)
 
-All objects participating in the split have the `Split` headers set. Depending on the place in the split hierarchy it has different field combinations. There are four possible cases:
+All objects participating in the split have the `Split` headers set. Depending on the place in the split hierarchy it has different field combinations. There are five possible cases:
+
+* Initial object \
+  Initial object that expresses a desire to put a large object in the NeoFS network. It has `split_id` and `parent_header` fields set. The parent header is not finished: since there is no information about payload, it cannot be measured, hashed and signed (also, no object ID can be assigned). However, it has all user information attached to the object (e.g. attributes), and plays the role of representative of the original object until all the parts are uploaded to the NeoFS..
 
 * First part \
-  First part object only has the `split_id` field set, as there is no more information known at this point
+  First part object only has the `split_id` and `init` fields set, as there is no more information known at this point
```

Comment:
So, do we need `split_id`?

Reply:
cant add any more thoughts except i've already written: https://github.com/nspcc-dev/neofs-api/pull/283#issuecomment-1917244465

Reply:
You have this `init` field in each and every piece. It can be used instead of `split_id`. Or not?

Reply:
oh, you mean using init object as a split ID? well, it should work. should a new SN generate split_id for backward compatibility?

Reply:
Link object is not compatible, so even if you add `split_id` old node couldn't reconstruct the object (at least without falling back to chain reconstruction).